modules.d/95nfs/module-setup.sh

```diff
@@ -120,8 +120,13 @@ install() {
     mkdir -m 0755 -p "$initdir/var/lib/nfs"
     mkdir -m 0755 -p "$initdir/var/lib/nfs/rpc_pipefs"
     mkdir -m 0770 -p "$initdir/var/lib/rpcbind"
-    [ -d "/var/lib/nfs/statd/sm" ] && mkdir -m 0755 -p "$initdir/var/lib/nfs/statd/sm"
-    [ -d "/var/lib/nfs/sm" ] && mkdir -m 0755 -p "$initdir/var/lib/nfs/sm"
+    [ -d "$dracutsysrootdir/var/lib/nfs/statd/sm" ] \
```

Comment:
This directory is not provided upstream, afaik it's RH/Fedora specific. See `man rpc.statd(8)` or http://git.linux-nfs.org/?p=steved/nfs-utils.git;a=blob;f=utils/statd/statd.man;h=7441ffde2687e2a541bdea78d04a30358cccb558;hb=refs/heads/master#l434